engine/framework/internal/worker/master_client.go

```diff
@@ -214,13 +237,15 @@ func (m *MasterClient) CheckMasterTimeout(clk clock.Clock) (ok bool, err error)
 }
 
 // SendHeartBeat sends a heartbeat to the master.
-func (m *MasterClient) SendHeartBeat(ctx context.Context, clock clock.Clock, isFinished bool) error {
+func (m *MasterClient) SendHeartBeat(ctx context.Context) error {
```

Comment:
Better to record more information in the log of L261, such as `epoch` and `sendTime`.